.github/workflows/ci-4.x.yml

```diff
@@ -33,6 +33,42 @@ jobs:
         with:
           name: heap-dump
           path: target/*.hprof
+  Test-ISPN14:
+    name: Run tests
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        jdk: [ 11 ]
+        profile: [ISPN-14]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ matrix.jdk }}
+          distribution: temurin
+      - name: Run tests
+        run: mvn -P ${{ matrix.profile }} -s .github/maven-ci-settings.xml -B -DtestLogLevel=OFF test
+  Test-ISPN14-jakarta:
+    name: Run tests
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        jdk: [ 11 ]
+        profile: [ ISPN-14-jakarta ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ matrix.jdk }}
+          distribution: temurin
+      - name: Run tests
+        run: mvn -P ${{ matrix.profile }} -s .github/maven-ci-settings.xml -B -DtestLogLevel=OFF test
```

Comment:
The Maven profile is the only difference between test job definitions, correct?

In this case, how about merging them and using matrix with [exclusions](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations) ?

Reply:
I had two different jobs, because one job was always failing due to the missing ISPN package. But now I've merged the two jobs into one. Is that what you meant?